reviews.Rmd

```diff
@@ -57,7 +61,8 @@ Type I and Type II error rates:
 * Our Response: The Bayesian rational considers theoretical risk and detectability simultaneously a priori. From a severity perspective, either Type I or Type II, never both, is considered post hoc, depending on the test results and if the research hypothesis is H1 or H2. 
```

Comment:
Please clarify this sentence "From a severity perspective,..." What is H1 or H2 here? What is posthoc here?

Reply:
```suggestion
* Our Response: The Bayesian rational considers theoretical risk and detectability simultaneously a priori. From a severity perspective, either Type I or Type II, never both, is considered post hoc (i.e., after a test result was obtained from data) depending on the test results and if the research hypothesis is H0 or H1. 
```